Sources/MockoloFramework/Templates/ThrowingKindTemplate.swift

```diff
@@ -15,13 +15,26 @@
 //
 
 extension ThrowingKind {
-    func applyThrowingTemplate() -> String? {
+    /// if this method is called to render handler for closure, rethrows should be replaced with throws.
+    ///
+    /// - example:
+    ///     ```
+    ///     // handler shouldn't use rethrows, instead use throws
+    ///     var fooHandler: (() -> throws -> Void) throws -> Void
+    ///     func foo(bar: () throws -> Void) rethrows -> Void
+    ///     ```
+    func applyThrowingTemplate(
+        appliesforClosureHandler: Bool
```

Comment:
I would like to avoid, as much as possible, structuring code where special cases branch out within a common process.
This would require everyone using this feature to understand the impact caused by the flag, and the internal code could become unnecessarily complex.
I believe you can see why this is complicated, given that this pattern is widespread in the existing code.

If you only want to change `rethrows` to `throws`, I think it is enough to simply coerce it to `.any` at the caller's site.

```swift
extension ThrowingKind {
    var coerceRethrowsToThrows: ThrowingKind {
        if case .rethrows = self {
            return .throws // renamed .any to .throws
        }
        return self
    }
}
```

Reply:
Got it. Thank you so much! I agree with you.

I applied it here: https://github.com/uber/mockolo/pull/262/commits/f1121102273819a4ff11f3c45142be10cde6abaf